frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -437,9 +439,11 @@ function getMeshItems(
                   ? "Cannot merge because the proofreading tool is not active."
                   : maybeUnmappedSegmentId == null
                     ? "The mesh wasn't loaded in proofreading mode. Please reload the mesh."
-                    : activeSegmentMissing
-                      ? "Select a segment first."
-                      : null
+                    : meshFileMapping != null
+                      ? "This mesh was created for a mapping. Please use a meshfile that is based on an oversegmentation."
```

Comment:
Maybe this is a more explicit / easier to understand message? In explicitly includes the word unmapped which might help users to understand that the meshfile should be computed on the "raw oversegmented" prediction data?
```suggestion
                      ? "This mesh was created for a mapping. Please use a meshfile that is based on an unmapped oversegmentation data."
```

@fm3 Maybe you have a better phrasing for this? Or do you think @dieknolle3333 message is already enough and mine it so detailed?

Reply:
(I know I am not florian but I feel like this is a good idea so I added the more detailed phrase :) )

Reply:
sounds good :)